utils/utils.py

```diff
@@ -303,14 +338,16 @@ def ind2rgb(arr, color):
 
 def list_input_images(img_dir_or_csv: str,
                       bucket_name: str = None,
-                      glob_patterns: List = None):
+                      glob_patterns: List = None,
+                      in_case_of_path: str = None):
     """
     Create list of images from given directory or csv file.
 
     :param img_dir_or_csv: (str) directory containing input images or csv with list of images
     :param bucket_name: (str, optional) name of aws s3 bucket
-    :param glob_patterns: (list of str) if directory is given as input (not csv), these are the glob patterns that will be used
-                        to find desired images
+    :param glob_patterns: (list of str) if directory is given as input (not csv),
+                           these are the glob patterns that will be used to find desired images
+    :param in_case_of_path: (str) directory that can contain the images if not the good one in the csv
```

Comment:
Why adding this `in_case_of_path`? Shouldn't user validate the existence of images in csv prior to inputting it? This is, for example, the purpose of a `validate_geodata.py` type of script, but can also be done manually (which is not ideal of course). 

Again, this is mistake-handling and refers back to my comment under inference_segmentation about state_dict mistake handling.